### ISSUE TITLE
Nunchuk Accelerometer should have their own axes as well.

### ISSUE THREAD

**Issue by jgcodes2020** [jloehr/HID-Wiimote#57]
The driver provides for 6 axes. I assume when a Wii U Pro Controller/Wii Classic Controller is connected, 4 of the axes go to the two sticks, with 2 left for accelerometers.

The nunchuk also has an accelerometer. If I connect a Wiimote + Nunchuk, the accelerometer axes of the nunchuck should show up as well.

**Comment by jloehr** [comment]
Yes, the Nunchuck has an accelerometer. However an accelerometer tracks change in motion, neither position nor orientation in space. So, mapping the raw accelerometer data to an axis doesn't bring any benefit.

Accelerometer can be used to detect shaking though. This could then be mapped to either a button, like one quick shake to trigger an action, or to a single axis to track the shake intensity for e.g. throttle.

**Comment by jloehr** [comment]
Oh, wait I think it does pick up gravity, so you can actually track its position on two axes. Can't remember why I didn't map the data 🤔

**Comment by MakkuSaiko** [comment]
Could always map nunchuck as a button, with the option of switching the feature off
